lib/app/features/auth/views/pages/discover_creators/discover_creators.dart

```diff
@@ -85,46 +83,41 @@ class DiscoverCreators extends HookConsumerWidget {
                   slivers: [
                     SliverPadding(padding: EdgeInsets.only(top: 34.0.s)),
                     ...slivers,
-                    SliverPadding(
-                      padding: EdgeInsets.only(
-                        bottom: 16.0.s + (mayContinue ? 0 : MediaQuery.paddingOf(context).bottom),
-                      ),
-                    ),
+                    SliverPadding(padding: EdgeInsets.only(bottom: 16.0.s)),
                   ],
                 );
               },
             ),
           ),
-          if (mayContinue)
-            Column(
-              children: [
-                const HorizontalSeparator(),
-                SizedBox(height: 16.0.s),
-                ScreenSideOffset.small(
-                  child: Button(
-                    disabled: finishNotifier.isLoading,
-                    trailingIcon: finishNotifier.isLoading ? const IONLoadingIndicator() : null,
-                    label: Text(context.i18n.button_continue),
-                    mainAxisSize: MainAxisSize.max,
-                    onPressed: () {
-                      ref.read(onboardingDataProvider.notifier).followees =
-                          selectedCreators.map((creator) => creator.pubkey).toList();
-
-                      guardPasskeyDialog(
-                        context,
-                        (child) => RiverpodPasskeyRequestBuilder(
-                          provider: onboardingCompleteNotifierProvider,
-                          request: () =>
-                              ref.read(onboardingCompleteNotifierProvider.notifier).finish(),
-                          child: child,
-                        ),
-                      );
-                    },
-                  ),
+          Column(
+            children: [
+              const HorizontalSeparator(),
+              SizedBox(height: 16.0.s),
+              ScreenSideOffset.small(
+                child: Button(
+                  disabled: finishNotifier.isLoading,
+                  trailingIcon: finishNotifier.isLoading ? const IONLoadingIndicator() : null,
+                  label: Text(context.i18n.button_continue),
+                  mainAxisSize: MainAxisSize.max,
+                  onPressed: () {
+                    ref.read(onboardingDataProvider.notifier).followees = [
```

Comment:
Consider using the `map` function here.

`ref.read(onboardingDataProvider.notifier).followees = selectedCreators.map((creator) => creator.pubkey).toList();`